InstallationGuide.md

```diff
@@ -155,9 +145,9 @@ The output should look similar to
 ```
 Apache Maven 3.6.3
 Maven home: /usr/share/maven
-Java version: 11.0.11, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
-Default locale: en, platform encoding: UTF-8
-OS name: "linux", version: "5.10.60.1-microsoft-standard-wsl2", arch: "amd64", family: "unix"
+Java version: 11.0.13, vendor: Ubuntu, runtime: /usr/lib/jvm/java-11-openjdk-amd64
+Default locale: es_ES, platform encoding: UTF-8
```

Comment:
Do we really want to define the standard to be "es_ES" in the official IDSA repo? Shouldn't we rather stay with "en" since english is the official language used?

Reply:
Modified to "en" because it is the official IDSA repository language.